ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm.robot

```diff
@@ -295,7 +299,7 @@ Verify User Can Validate Scale To Zero
     Wait For Pods To Be Terminated    label_selector=serving.kserve.io/inferenceservice=${flan_model_name}
     ...    namespace=autoscale-zero
     [Teardown]   Clean Up Test Project    test_ns=autoscale-zero
-    ...    isvc_names=${model_name}
+    ...    isvc_names=${model_name}    wait_prj_deletion=${FALSE}
```

Comment:
Don't you want to rename this as you did for the others around so it's consistent?
```suggestion
    ...    isvc_names=${models_names}    wait_prj_deletion=${FALSE}
```

Reply:
done!